pkg/util/limiter/query_limiter.go

```diff
@@ -98,6 +102,17 @@ func (ql *QueryLimiter) AddChunkBytes(chunkSizeInBytes int) error {
 	return nil
 }
 
+// AddDataBytes adds the input label size in bytes and returns an error if the limit is reached.
```

Comment:
do you mean `adds queried data size in bytes` instead of `input label size`?

Reply:
Addressed this in latest revision.